jax/_src/numpy/ufuncs.py

```diff
@@ -2553,11 +2553,14 @@ def divmod(x1: ArrayLike, x2: ArrayLike, /) -> tuple[Array, Array]:
 def _float_divmod(x1: ArrayLike, x2: ArrayLike) -> tuple[Array, Array]:
   # see float_divmod in floatobject.c of CPython
   mod = lax.rem(x1, x2)
+  mod = _where(logical_and(abs(x2) == np.inf, abs(x1) != np.inf), x1, mod)
```

Comment:
`isinf` would be better here:
```suggestion
  mod = _where(isinf(x2) & ~isinf(x1), x1, mod)
```

Reply:
Done.